File: Parking Lot/Model/Car.cs
namespace Model
{
    public class Car
    {
        public string? RegistrationNumber { get; set; }
        public string? Color { get; set; }

        public Car(string registrationNumber, string color)
        {
            RegistrationNumber = registrationNumber;
            Color = color;
        }
    }
}

File: design patterns/Creational/Builder Design pattern/src/Car.java
import java.awt.*;

public class Car {

    public Car(CarBuilder carBuilder) {
        this.bodyType = carBuilder.bodyType;
        this.color = carBuilder.color;
        this.fuelType = carBuilder.fuelType;
        this.transmission = carBuilder.transmission;
    }

    private String color;

    @Override
    public String toString() {
        return "Car{" +
                "color='" + color + '\'' +
                ", fuelType='" + fuelType + '\'' +
                ", transmission='" + transmission + '\'' +
                ", bodyType='" + bodyType + '\'' +
                '}';
    }

    private String fuelType;

    private String transmission;

    private String bodyType;
}
